src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -856,6 +883,8 @@ class device {
              << std::setw(16) << sensor_tree::get_pretty<unsigned short>( "board.physical.thermal.tcrit_temp")
              << std::setw(16) << sensor_tree::get<std::string>( "board.physical.thermal.fan_presence")
              << std::setw(16) << sensor_tree::get_pretty<unsigned short>( "board.physical.thermal.fan_speed" ) << std::endl;
+        ostr << std::setw(16) << "HBM Temp" << std::endl;
```

Comment:
memory temp are displayed next to the mem banks. we probably should do the same for hbm, so there should be no change here.

Reply:
Done.